pkg/daprrp/api/v20231001preview/configurationstore_conversion.go

```diff
@@ -73,7 +73,7 @@ func (src *DaprConfigurationStoreResource) ConvertTo() (v1.DataModelInterface, e
 		if src.Properties.Recipe != nil && (!reflect.ValueOf(*src.Properties.Recipe).IsZero()) {
 			msgs = append(msgs, "recipe details cannot be specified when resourceProvisioning is set to manual")
 		}
-		if src.Properties.Metadata == nil || len(src.Properties.Metadata) == 0 {
+		if len(src.Properties.Metadata) == 0 {
```

Comment:
`golangci-lint` was complaining about this saying that first check was unnecessary.